decidim-meetings/app/assets/javascripts/decidim/meetings/admin/meetings_form.js.es6

```diff
@@ -96,6 +96,30 @@
     toggleDisabledHiddenFields();
 
     attachGeocoding($form.find("#meeting_address"));
+
+    const $meetingRegistrationType = $form.find("#meeting_registration_type");
+    const $meetingRegistrationTerms = $form.find("#meeting_registration_terms");
+    const $meetingRegistrationUrl = $form.find("#meeting_registration_url");
+    const $meetingAvailableSlots = $form.find("#meeting_available_slots");
+
+    const toggleDependsOnSelect = ($target, $showDiv, type) => {
+      const value = $target.val();
+      $showDiv.hide();
+      if (value === type) {
+        $showDiv.show();
+      }
+    };
```

Comment:
There is a `toggle()` function that can simplify this 😄 https://api.jquery.com/toggle/

Reply:
yes, this is nice :)